src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="styles/images/imac.jpeg"
+        alt="Product image"
+        class="card__product-image"
+      >
+
+      <h2 class="card__product-title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="review card__review">
+        <div class="review__stars review__stars--rating_4">
+          <div class="review__single-star"></div>
+          <div class="review__single-star"></div>
+          <div class="review__single-star"></div>
+          <div class="review__single-star"></div>
+          <div class="review__single-star"></div>
+        </div>
+
+        <p class="review__review-number">
+          Reviews: 5
+        </p>
+      </div>
```

Comment:
You shouldn't create extra BEM blocks, use only card and stars from previous task